libs/core/schematics/ng-add/index.ts

```diff
@@ -67,34 +80,31 @@ function addDependencies(options: Schema): Rule {
             });
         }
 
-        if (options.styleFonts) {
-            if (
-                !hasPackage(tree, 'fundamental-styles') ||
-                checkPackageVersion(tree, 'fundamental-styles', 'FDSTYLES_VER_PLACEHOLDER', '<')
-            ) {
-                dependencies.push({
-                    type: NodeDependencyType.Default,
-                    // Will be replaced with the real version during sync-version script run
-                    version: `FDSTYLES_VER_PLACEHOLDER`,
-                    name: 'fundamental-styles',
-                    overwrite: true
-                });
-            }
-
-            if (
-                !hasPackage(tree, '@sap-theming/theming-base-content') ||
-                checkPackageVersion(tree, '@sap-theming/theming-base-content', 'THEMING_VER_PLACEHOLDER', '<')
-            ) {
-                dependencies.push({
-                    type: NodeDependencyType.Default,
-                    // Will be replaced with the real version during sync-version scipt run
-                    version: `THEMING_VER_PLACEHOLDER`,
-                    name: '@sap-theming/theming-base-content',
-                    overwrite: true
-                });
-            }
+        if (
```

Comment:
If you mean make styles & base-theming deps of the ngx/core - no we can't.

In the code below we set some assets & styles using relative paths 

```
./node_modules/@sap-theming/theming-base-content/content/Base/baseLib/
```

In case we will have them as deps they may be placed under the specified path or under

```
./node_modules/@fundamental-ngx/core/node_modules/@sap-theming/theming-base-content/content/Base/baseLib/
```

and it may change in future, see more about in ticket https://github.com/SAP/fundamental-ngx/issues/8350